### PR TITLE
[Improvement](docs) refine query acceleration doc

### PR DESCRIPTION
**Pull request by xzj7019**
## Versions 

- [x] dev
- [ ] 3.0
- [ ] 2.1
- [ ] 2.0

## Languages

- [x] Chinese
- [x] English

## Docs Checklist

- [ ] Checked by AI
- [ ] Test Cases Built

### CODE REVIEW

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/colocation-join.md [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+PROPERTIES(
+    "colocate_with" = "group1"
+);
+```
+
+如果指定的 Group 不存在，则 Doris 会自动创建一个只包含当前这张表的 Group。如果 Group 已存在，则 Doris 会检查当前表是否满足 Colocation Group Schema。如果满足，则会创建该表，并将该表加入 Group。同时，表会根据已存在的 Group 中的数据分布规则创建分片和副本。Group 归属于一个 Database，Group 的名字在一个 Database 内唯一。在内部存储是 Group 的全名为 `dbId_groupName`，但用户只感知 groupName。
+
+
+
+:::tip
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/colocation-join.md:94]: :::tip 提示

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/colocation-join.md [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+2.0 版本中，Doris 支持了跨 Database 的 Group。
+:::
+
+在建表时，需使用关键词 `__global__` 作为 Group 名称的前缀。如：
+
+```
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/colocation-join.md:100]: add sql

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/colocation-join.md [hunk 3]
```diff
@@ -0,0 +1,445 @@
...
+| 6           | 10003, 10004, 10001 |
+| 7           | 10003, 10004, 10002 |
++-------------+---------------------+
+```
+
+- BucketIndex：分桶序列的下标。
+
+- BackendIds：分桶中数据分片所在的 BE 节点 id 列表。
+
+:::note
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/colocation-join.md:170]: :::info 备注

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/colocation-join.md [hunk 4]
```diff
@@ -0,0 +1,445 @@
...
+
+### 副本修复
+
+副本只能存储在指定的 BE 节点上。所以当某个 BE 不可用时（宕机、Decommission 等），需要寻找一个新的 BE 进行替换。Doris 会优先寻找负载最低的 BE 进行替换。替换后，该 Bucket 内的所有在旧 BE 上的数据分片都要做修复。迁移过程中，Group 被标记为 Unstable。
+
+### 副本均衡
+
+Doris 会尽力将 Colocation 表的分片均匀分布在所有 BE 节点上。对于普通表的副本均衡，是以单副本为粒度的，即单独为每一个副本寻找负载较低的 BE 节点即可。而 Colocation 表的均衡是 Bucket 级别的，即一个 Bucket 内的所有副本都会一起迁移。我们采用一个简单的均衡算法，即在不考虑副本实际大小，而只根据副本数量，将 BucketsSequence 均匀的分布在所有 BE 上。具体算法可以参阅 `ColocateTableBalancer.java` 中的代码注释。
+
+:::caution
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/colocation-join.md:210]: :::caution 注意

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/performance-tuning-overview/diagnostic-tools.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+高效好用的性能诊断工具对于数据库系统的调优至关重要，因为这关系到是否能快速定位到存在性能问题的业务 SQL，继而快速定位和解决性能瓶颈，保证数据库系统服务的 SLA。
+
+当前，Doris 系统默认将执行时间超过 5 秒的 SQL 认定为慢 SQL，此阈值可通过 `config.qe_slow_log_ms` 进行配置。目前 Doris 提供了以下三种诊断渠道，能够帮助快速定位存在性能问题的慢 SQL，分别如下：
+
+## Doris Manager 日志
+
+Doris Manager 的日志模块提供了慢 SQL 筛选功能。用户可以通过选择特定 FE 节点上的 `fe.audit.log` 来查看慢 SQL。只需在搜索框中输入“slow_query”，即可在页面上展示出当前系统的历史慢 SQL 信息，如下图所示：
+
+![](static/EbJobMOIso1vIgx5mYgcXTRrnYc.png)
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/performance-tuning-overview/diagnostic-tools.md:37]: src - static - images. link should be `![](/images/XX`

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/tuning/tuning-plan/accelerating-queries-with-sql-cache.md [hunk 6]
```diff
@@ -24,67 +24,14 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-## 工作原理
+## 概述
 
-关于 SQL Cache 详细实现原理，请参考 [查询缓存（SQL Cache）](../../../query-acceleration/sql-cache-manual)
+关于 SQL Cache 详细实现原理，请参考 [查询缓存（SQL Cache）](https://selectdb.feishu.cn/wiki/GhRhwLOXNi5muzkb5PacLR4Wn6e)
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/tuning/tuning-plan/accelerating-queries-with-sql-cache.md:29]: (../query-acceleration/sql-cache-manual）

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/tuning/tuning-plan/adjusting-join-shuffle.md [hunk 7]
```diff
@@ -24,40 +24,28 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-## 工作原理
+## 概述
 
 Doris 支持使用 Hint 来调整 Join 操作中数据 Shuffle 的类型，从而优化查询性能。本节将详细介绍如何在 Doris 中利用 Hint 来指定 Join Shuffle 的类型。
 
-目前，Doris 仅限于指定 Join 右表的 Distribute Type，并且仅提供两种类型供选择：`[shuffle] `和 `[broadcast]`。Distribute Type 需置于 Join 右表之前，可采用中括号 `[]`的方式。
+目前，Doris 支持两种独立的 [DistributeHint](https://selectdb.feishu.cn/wiki/BpBxwpu07iL4jjkeai1cfs24nrn?open_in_browser=true)，`[shuffle] ` 和 `[broadcast]`，用来指定 Join 右表的 Distribute Type。Distribute Type 需置于 Join 右表之前，采用中括号 `[]` 的方式。同时，Doris 也可以通过 LeadingHint 配合 DistributeHint 的方式，指定 shuffle 方式（详见“使用 Leading Hint 控制 Join 顺序”章节相关介绍）。
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/tuning/tuning-plan/adjusting-join-shuffle.md:31]: (../../query-acceleration/hint/sql-cache-manual)

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/tuning/tuning-plan/dml-tuning-plan.md [hunk 8]
```diff
@@ -24,100 +24,6 @@ specific language governing permissions and limitations
...
-
-- 位置：在会话级别进行设置
-
-- 作用：控制是否实际启用 Pipeline 引擎
-
-## 查询部分
-
-详细请参考[计划调优 - 其他章节](../../../query-acceleration/tuning/tuning-plan/optimizing-table-schema)
\ No newline at end of file
+Doris 支持从多种数据源导入数据，灵活运用 Doris 提供的多种导入功能，可以高效地将各种来源的数据导入到 Doris 中进行分析。最佳实践详情请参考[导入概览](https://doris.apache.org/zh-CN/docs/dev/data-operate/import/load-manual/)。
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/tuning/tuning-plan/dml-tuning-plan.md:29]: ../../data-operate/import/load-manual

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/tuning/tuning-plan/reordering-join-with-leading-hint.md [hunk 9]
```diff
@@ -24,174 +24,73 @@ specific language governing permissions and limitations
...
 -->
 
-## Leading Hint 简介
+## 概述
 
-Leading Hint 是一种强大的查询优化技术，它允许用户指导 Doris 优化器确定查询计划中的表连接顺序。正确使用 Leading Hint 可以显著提升复杂查询的性能。本文将详细介绍如何在 Doris 中使用 Leading Hint 来控制 Join 的顺序。
+Doris 支持使用 Hint 来调整 Join 操作中数据 Shuffle 的类型，从而优化查询性能。本节将详细介绍如何在 Doris 中利用 Hint 来指定 Join Shuffle 的类型。
 
-:::info 备注
+目前，Doris 支持两种独立的 [DistributeHint](https://selectdb.feishu.cn/wiki/BpBxwpu07iL4jjkeai1cfs24nrn?open_in_browser=true)，`[shuffle] ` 和 `[broadcast]`，用来指定 Join 右表的 Distribute Type。Distribute Type 需置于 Join 右表之前，采用中括号 `[]` 的方式。同时，Doris 也可以通过 LeadingHint 配合 DistributeHint 的方式，指定 shuffle 方式（详见“使用 Leading Hint 控制 Join 顺序”章节相关介绍）。
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/tuning/tuning-plan/reordering-join-with-leading-hint.md:31]: ../../query-acceleration/hints/distribute-hint

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/tuning/tuning-plan/transparent-rewriting-with-sync-mv.md [hunk 10]
```diff
@@ -24,32 +24,28 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-## 工作原理
+## 概述
 
-单表物化视图 (Materialized View) 是一种特殊的表，它预先根据定义好的 SELECT 语句计算并存储数据。其主要目的是满足用户对原始明细数据的任意维度分析需求，同时也能快速地进行固定维度的分析查询。
+[单表物化视图](https://selectdb.feishu.cn/wiki/KOOYwALDdiu5ACk1UdMcKS9Tn1c?open_in_browser=true) （Materialized View） 是一种特殊的表，它预先根据定义好的 SELECT 语句计算并存储数据。其主要目的是满足用户对原始明细数据的任意维度分析需求，同时也能快速地进行固定维度的分析查询。
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/tuning/tuning-plan/transparent-rewriting-with-sync-mv.md:29]: ../../query-acceleration/materialized-view/sync-materialized-view

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/colocation-join.md (hunk 1 above)
Comment (xzj7019) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/colocation-join.md]: done

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/colocation-join.md (hunk 2 above)
Comment (xzj7019) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/colocation-join.md]: done

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/colocation-join.md (hunk 3 above)
Comment (xzj7019) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/colocation-join.md]: done

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/colocation-join.md (hunk 4 above)
Comment (xzj7019) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/colocation-join.md]: done

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/performance-tuning-overview/diagnostic-tools.md (hunk 5 above)
Comment (xzj7019) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/performance-tuning-overview/diagnostic-tools.md]: done

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/tuning/tuning-plan/adjusting-join-shuffle.md (hunk 7 above)
Comment (xzj7019) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/tuning/tuning-plan/adjusting-join-shuffle.md]: done

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/tuning/tuning-plan/reordering-join-with-leading-hint.md (hunk 9 above)
Comment (xzj7019) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/tuning/tuning-plan/reordering-join-with-leading-hint.md]: done

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/tuning/tuning-plan/transparent-rewriting-with-sync-mv.md (hunk 10 above)
Comment (xzj7019) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/tuning/tuning-plan/transparent-rewriting-with-sync-mv.md]: done

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/tuning/tuning-plan/accelerating-queries-with-sql-cache.md (hunk 6 above)
Comment (xzj7019) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/tuning/tuning-plan/accelerating-queries-with-sql-cache.md]: done

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/tuning/tuning-plan/dml-tuning-plan.md (hunk 8 above)
Comment (xzj7019) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/query-acceleration/tuning/tuning-plan/dml-tuning-plan.md]: done

**Pull request merged**